pom.xml

```diff
@@ -122,6 +122,11 @@
             <version>1.7.30</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.12</artifactId>
+            <version>2.7.0</version>
+        </dependency>
```

Comment:
Should this be set as `provided`? You expect to run this from Kafka where the Kafka JARs are already available.

Reply:
Yes this should be changed. We plan to leverage what Kafka is already serving